Sources/EdgeNetworkHandlers/EdgeEventWarning.swift

```diff
@@ -29,21 +29,16 @@ struct EdgeEventWarning: Codable {
     /// The warning report
     let report: EdgeEventWarningReport?
 
-    /// Encodes the event to which this warning is attached as the index in the events array in EdgeRequest
-    let eventIndex: Int?
-
     // MARK: - Codable
     enum CodingKeys: String, CodingKey {
         case type
         case status
         case title
         case report
-        case eventIndex
     }
 
     public func encode(to encoder: Encoder) throws {
         var container = encoder.container(keyedBy: CodingKeys.self)
-        // skip eventIndex when encoding
```

Comment:
Looks like we used to skip the eventIndex when encoding as it was not useful for extensions listening to Edge responses. Can we do the same for the EdgeEventWarningReport and EdgeErrorReport?
We already cover with unit tests that the decoding works as expected and report.eventIndex is set.